lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -132,17 +135,16 @@ def prepare_data(d, init):
     has_curve = bool(curves)
 
     # Get ScalarMappables.
-    mappabledict = {}
+    mappabledict = []
```

Comment:
```suggestion
    labeled_mappables = []
```
Adapt the variable name to the changed semantics and make it similar to the lines case.